client-core/src/signer/dummy_signer.rs

```diff
@@ -25,7 +25,7 @@ impl DummySigner {
         let unit = 16_usize;
         let plain_payload_len = plain_txaux.encode().len();
         // PKCS7 padding -- if the len is multiple of the block length, it'll still be padded
-        vec![0; plain_payload_len + unit - plain_payload_len % unit]
+        vec![0; plain_payload_len + unit - plain_payload_len % unit + 5]
     }
```

Comment:
rfc8452 doesn't seem to have this padding -- seems the length should be `plain_payload_len + tag_len(16)`

Reply:
as Aad is txid, fixed to 32

Reply:
> as Aad is txid, fixed to 32

Yeah, `plain_len + 32` passes the test too, = ="